app.js

```diff
@@ -8,6 +8,8 @@ var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
 var postsRouter = require('./routes/posts');
+var editRouter = require('./routes/posts');
+var deleteRouter = require('./routes/posts');
```

Comment:
you should export the individual functions them selves isntead of the whole file itself. Check comment below

Reply:
Correction, should import the router file once.